Makefile

```diff
@@ -354,45 +356,32 @@ $(MAC_BINARIES_ZIP_FILE): $(DIST_DIR) $(MAC_APPLICATION_BUNDLE) $(USERGUIDE_TARG
 build/vcredist_x86.exe: | build
 	powershell.exe -Command "Start-BitsTransfer -Source https://download.microsoft.com/download/5/D/8/5D8C65CB-C849-4025-8E95-C3966CAFD8AE/vcredist_x86.exe -Destination build\vcredist_x86.exe"
 
-CERT_FILE := codesigning-2021.crt
-KEY_FILE := Stanford-natcap-code-signing-cert-expires-2024-01-26.key.pem
-P12_FILE := Stanford-natcap-code-signing-cert-expires-2024-01-26.p12
 KEYCHAIN_NAME := codesign_keychain
 codesign_mac:
 	# download the p12 certificate file from google cloud
-	$(GSUTIL) cp 'gs://stanford_cert/$(P12_FILE)' '$(BUILD_DIR)/$(P12_FILE)'
+	$(GSUTIL) cp gs://stanford_cert/$(CERT_FILE) $(BUILD_DIR)/$(CERT_FILE)
 	# create a new keychain (so that we can know what the password is)
-	security create-keychain -p '$(KEYCHAIN_PASS)' '$(KEYCHAIN_NAME)'
+	security create-keychain -p $(KEYCHAIN_PASS) $(KEYCHAIN_NAME)
 	# add the keychain to the search list so it can be found
-	security list-keychains -s '$(KEYCHAIN_NAME)'
+	security list-keychains -s $(KEYCHAIN_NAME)
 	# unlock the keychain so we can import to it (stays unlocked 5 minutes by default)
-	security unlock-keychain -p '$(KEYCHAIN_PASS)' '$(KEYCHAIN_NAME)'
+	security unlock-keychain -p $(KEYCHAIN_PASS) $(KEYCHAIN_NAME)
 	# add the certificate to the keychain
 	# -T option says that the codesign executable can access the keychain
 	# for some reason this alone is not enough, also need the following step
-	security import $(BUILD_DIR)/$(P12_FILE) -k '$(KEYCHAIN_NAME)' -P '$(CERT_KEY_PASS)' -T /usr/bin/codesign
+	security import $(BUILD_DIR)/$(CERT_FILE) -k $(KEYCHAIN_NAME) -P "$(CERT_PASS)" -T /usr/bin/codesign
 	# this is essential to avoid the UI password prompt
-	security set-key-partition-list -S apple-tool:,apple: -s -k '$(KEYCHAIN_PASS)' '$(KEYCHAIN_NAME)'
-	# sign the dmg using certificate that's looked up by unique identifier 'Stanford Univeristy'
-	codesign --timestamp --verbose --sign 'Stanford University' $(MAC_DISK_IMAGE_FILE)
+	security set-key-partition-list -S apple-tool:,apple: -s -k $(KEYCHAIN_PASS) $(KEYCHAIN_NAME)
+	# sign the dmg using certificate that's looked up by unique identifier 'Stanford'
+	codesign --timestamp --verbose --sign Stanford $(MAC_DISK_IMAGE_FILE)
 	# relock the keychain (not sure if this is important?)
-	security lock-keychain '$(KEYCHAIN_NAME)'
+	security lock-keychain $(KEYCHAIN_NAME)
```

Comment:
I don't think this is necessary either.

Reply:
Hey Emily, I was searching around trying to learn some things and came across this post: https://stackoverflow.com/questions/57838889/manage-keychain-to-codesign-macos-ios-app-with-xcodebuild-unattended.

Would it make sense to delete the keychain instead of locking it here?

```
# clean up temp keychain we created
security delete-keychain ${tempKeychain}
```

Reply:
I'm not sure if that is needed either... I'm guessing they suggest that because they're running it locally on their own computer, in which case you wouldn't want stray keychains cluttering things up. On github actions, having that keychain stick around until the end of the workflow should have no effect.

Reply:
I see, thanks. So it might just be a choice between letting things get garbage collected or handling the procedure full circle ourselves. If there is seemingly a standard set of operations I guess I'd lean to explicitly doing them, even if it might be redundant or unnecessary. What do you think?

Reply:
Since we're now not keeping the password a secret, I'd think there's no reason at all to lock the keychain afterward.